docs/TrialAccounts/offboarding.md

```diff
@@ -0,0 +1,180 @@
+---
+sidebar_label: 'Offboarding Trial Accounts'
+---
+# Introduction
+
+Trial Accounts are a great way to onboard new users to applications with the click of a link. The in-app experience is seamless but at some point, the trial must end. This process should remove all restrictions from the user's account and allow them to interact with the any app on NEAR. This process is known as offboarding and is a critical part of the user's journey.
+
+As part of the offboarding process, a full access key will be added to the account and this should be stored somewhere safe. For this reason, we wanted to make sure trial accounts could be exited into any wallet or app that supports the official linkdrop standard (e.g MyNEARWallet / FastAuth).
+
+The desired user experience is as follows:
+1. Click a link or scan a QR code.
+2. Enter a username.
+3. Immediately get signed into the app and start using it.
+4. Once your trial is over, a modal pops up with options for onboarding.
+5. Click the desired option (such as MyNEARWallet) and get redirected.
+6. Offboard with the app and get a new private key based on the option you chose (seedphrase, biometrics etc.)
+7. You're now a full, unrestricted NEAR account that can interact with the rest of the ecosystem.
```

Comment:
I don't like how the perspective has shifted here to "You are now a full, unrestricted NEAR account" after being objective the whole way through

Maybe "The trial account has now been converted into a full, unrestricted etc."